src/celeritas/field/FieldDriver.hh

```diff
@@ -81,11 +82,11 @@ class FieldDriver
 
     //// HEPER FUNCTIONS ////
 
-    // Find the next acceptable chord of with the miss-distance
+    // Find the next acceptable chord whose sagitta is less than delta_chord
```

Comment:
Just for a comment:  the miss-distrance here is the closest distance between the mid-point (from a stepper) and the chord, so it is not exactly sagitta (even though it mimics somewhat very close meaning), which is the reason  why the term sagitta is not directly used (in Geant4 as far as I know).

Reply:
Ah, ok, so it's an approximation but with the intent of being the "sagitta"? So approximately "the largest distance we can miss a boundary crossing" sort of?